instrumental/drivers/motion/_kinesis/isc.py

```diff
@@ -73,22 +73,44 @@ def _start_polling(self, polling_period='200ms'):
         self.polling_period = polling_period
         self.dev.StartPolling(self.polling_period.m_as('ms'))
 
+    def get_info(self):
```

Comment:
This seems unnecessary. Can you use something like `ffi.string()` instead of adding this new `decode` method?